AFQ/tests/test_data.py

```diff
@@ -1,9 +1,154 @@
-
+import botocore
+import filecmp
 import numpy as np
 import numpy.testing as npt
+import os
+import os.path as op
+import pytest
+import s3fs
+import shutil
+
+from glob import glob
+from moto import mock_s3
+from uuid import uuid4
 
 import AFQ.data as afd
 
+DATA_PATH = op.join(op.abspath(op.dirname(__file__)), "data")
+TEST_BUCKET = "test-bucket"
+TEST_DATASET = "ds000102-mimic"
+
+
+@pytest.fixture
+def temp_data_dir():
```

Comment:
Is this better in some way than https://github.com/yeatmanlab/pyAFQ/blob/master/AFQ/tests/test_api.py#L217 ?

Reply:
Oh, cool. I didn't know about `nibabel.tmpdirs`. The main difference I see is that `nbtmp.InTemporaryDirectory()` also changes the current working directory. Also, using is as a `pytest.fixture` with a yield statement will make sure that we delete the directory even if the test fails. I'm not sure what happens to the temporary directory if the test you linked in `test_AFQ_data` fails in the middle. but maybe it also deletes the directory.